src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.cpp

```diff
@@ -803,8 +803,8 @@ void FlightTaskAuto::_updateTrajConstraints()
 	if (_is_emergency_braking_active) {
 		// When initializing with large velocity, allow 1g of
 		// acceleration in 1s on all axes for fast braking
-		_position_smoothing.setMaxAcceleration({9.81f, 9.81f, 9.81f});
-		_position_smoothing.setMaxJerk(9.81f);
+		_position_smoothing.setMaxAcceleration({9.80665f, 9.80665f, 9.80665f});
```

Comment:
Please re-use the constant. 

https://github.com/PX4/PX4-Autopilot/blob/a6fcb8ef1ed123143314c96048a833d2937157a9/src/lib/geo/geo.h#L56

```suggestion
		_position_smoothing.setMaxAcceleration({CONSTANTS_ONE_G, CONSTANTS_ONE_G, CONSTANTS_ONE_G});
```